setup.sh

```diff
@@ -19,6 +19,12 @@ scram project ${CMSSWVER}
 cd ${CMSSWVER}/src/
 # cmsenv
 eval `scramv1 runtime -sh`
+echo "Needs your CERN username and pass word: NNKit is being cloned from gitlab"
+git clone https://gitlab.cern.ch/DeepAK8/NNKit.git
+cp NNKit/misc/mxnet_predict.xml $CMSSW_BASE/config/toolbox/$SCRAM_ARCH/tools/selected
+scram setup mxnet_predict
+cp NNKit/misc/lib/libmxnet_predict.so $CMSSW_BASE/external/$SCRAM_ARCH/lib/libmxnet_predict.so
+
 git cms-init
```

Comment:
you need to do `git cms-init` before doing anything else in the `$CMSSW_BASE/src` directory